presto-main/src/main/java/com/facebook/presto/cost/TableScanStatsRule.java

```diff
@@ -64,7 +63,7 @@ public TableScanStatsRule(Metadata metadata, StatsNormalizer normalizer)
         for (Map.Entry<Symbol, ColumnHandle> entry : node.getAssignments().entrySet()) {
             Symbol symbol = entry.getKey();
             Optional<ColumnStatistics> columnStatistics = Optional.ofNullable(tableStatistics.getColumnStatistics().get(entry.getValue()));
-            outputSymbolStats.put(symbol, columnStatistics.map(statistics -> toSymbolStatistics(tableStatistics, statistics)).orElse(UNKNOWN_STATS));
+            outputSymbolStats.put(symbol, columnStatistics.map(statistics -> toSymbolStatistics(tableStatistics, statistics)).orElse(SymbolStatsEstimate.unknown()));
```

Comment:
orElseGet

Reply:
Is there any practical difference in doing this? `SymbolStatsEstimate.unknown()` is a constant, and always be a constant. Why do we need to generate additional bootstrap methods for invokedynamic to pass it as a lambda?